indico/htdocs/sass/themes/indico.scss

```diff
@@ -560,7 +560,6 @@ div.event-details {
 
     .event-details-row {
         display: flex;
-        align-items: baseline;
```

Comment:
I don't see how this fixes anything. In my opinion, it brakes the alignment. To demonstrate:

With `align-items: baseline` the title is aligned in the same row as the content:
![image](https://user-images.githubusercontent.com/1579899/28717606-cb082f76-73a2-11e7-96da-6d8fe4ac1995.png)


With your fix:
![image](https://user-images.githubusercontent.com/1579899/28717667-069b99ba-73a3-11e7-949d-00cc75e1cf3d.png)